jenkins/job-configs/kubernetes-jenkins-pull/global.yaml

```diff
@@ -1,13 +1,28 @@
+- builder:
+    name: ensure-upload-to-gcs-script
+    builders:
+      - shell: |
+          _script_tmp_path='./_tmp/upload-to-gcs.sh'
+          _script_repo_path='hack/jenkins/upload-to-gcs.sh'
+          mkdir -p ./_tmp
+          if [[ "${{ghprbTargetBranch:-}}" == 'master' && -x "./kubernetes/${{_script_repo_path}}" ]]; then
```

Comment:
I feel like the second condition shouldn't be there. The only way that will be not true is if someone deleted the script, and we should call that a failure.